wrapper.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+import subprocess
+
+parser = argparse.ArgumentParser(description="SHM_CSR")
+parser.add_argument("input", help='Path of the input file')
+parser.add_argument('-m', '--method', help='Specify the method for the run: default= custom', default='custom')
+parser.add_argument('result_html', help='Path of the output file')
+parser.add_argument('result_dir', help='Path where the results will be stored')
+parser.add_argument('title', help='Specify the title for the run')
+parser.add_argument('-if', '--inc-fr', help='including fr1. Default = - ', default="-")
```

Comment:
What  are the available options here? Were you able to figure that out?
It looks like it has something to do with the inclusion of the FR1 region. 
Maybe this is an argument that is skipped  in `wrapper.sh`? Not all arguments are used I think.